src/DeveloperTeamLead.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Scanner;
+
+public class DeveloperTeamLead extends DeveloperEmployee {
+	private int developerExperience;
```

Comment:
add some validation here

Reply:
I added the validation for this attribute.